modules/core/src/test/java/org/apache/ignite/internal/processors/cache/persistence/snapshot/dump/IgniteCacheDumpSelfTest.java

```diff
@@ -146,6 +146,27 @@ public void testCacheDump() throws Exception {
         }
     }
 
+    /** */
+    @Test
+    public void testZippedCacheDump() throws Exception {
+        snpPoolSz = 4;
+
+        try {
+            IgniteEx ign = startGridAndFillCaches();
+
+            createDump(ign, DMP_NAME, null, true);
+
+            checkDump(ign);
+
+            createDump(cli, DMP_NAME + 2, null, true);
```

Comment:
This test doesn't check that client actually zips data. It just read dumped data, and info about compression extracted by reader itself. Let's add check for compression in 321 line.